build/docker/passwd

```diff
@@ -0,0 +1 @@
+ferretdb:x:10001:10001::/nonexistent:/nonexistent
```

Comment:
Please add a single-line break there

Reply:
Added a line break. https://github.com/FerretDB/FerretDB/pull/4004/commits/1bc653e83acca8ac66f33073b8790a9052f898c5

And as far as I know `GID` field can't be empty in `/etc/passwd`.

Reply:
My point was about adding `/etc/groups` and group ID into `USER` directory. I'm not sure if we need it

Reply:
I don't think it would be a problem for running FerretDB but I've found that Red Hat Enterprise Linux 7 Security Technical Implementation Guide suggest that the corresponding group ID should be provided in `/etc/groups`.

https://www.stigviewer.com/stig/red_hat_enterprise_linux_7/2017-07-08/finding/V-72003

According to that, I created a `group` file.